### PR TITLE
Add systemd tests 26-34, extend timeout for test 1, 15 and 16, remove binary tests summary needle check

### PR DESCRIPTION
**Pull request by eliroca**
Currently only tested for Tumbleweed x86_64

- Verification run: http://emerson.suse.de/tests/2150

- Testsuite package submitted: https://build.opensuse.org/request/show/798910

### CODE REVIEW

**Comment by eliroca** [comment]
@SergioAtSUSE the CI passed only after pushing the last [commit](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/10141/commits/cb34ded67e519624218a25dfd03b0d4901d3d5c0) where I add some quotation marks. Should I leave the last commit here, or does the CI need to be updated?

File: schedule/systemd/suse_patches-systemd_testsuite.yaml [hunk 1]
```diff
@@ -29,6 +29,15 @@ conditional_schedule:
...
                 - systemd_testsuite/test_23_type_exec
+                - systemd_testsuite/test_26_setenv
+                - systemd_testsuite/test_27_stdoutfile
+                - systemd_testsuite/test_28_percentj_wantedby
+                - systemd_testsuite/test_29_udev_id_renaming
+                - systemd_testsuite/test_30_onclockchange
+                - systemd_testsuite/test_31_device_enumeration
+                - systemd_testsuite/test_32_oompolicy
+                - systemd_testsuite/test_33_clean_unit
+                - systemd_testsuite/test_34_dynamicusermigrate
```
Comment (SergioAtSUSE) [inline schedule/systemd/suse_patches-systemd_testsuite.yaml:40]: If this is only for Tumbleweed, the conditional name `systemd_testsuite_15SP2ORLATER` is misleading.
Let's find a better name. What do you think of `systemd_testsuite_if_VERSION`?

**Comment by SergioAtSUSE** [comment]
> @SergioAtSUSE the CI passed only after pushing the last commit where I add some quotation marks. Should I leave the last commit here, or does the CI need to be updated?
> EDIT: Quotation marks were added: #10152

I suppose this was already fixed by rebasing, right?

File: schedule/systemd/suse_patches-systemd_testsuite.yaml (hunk 1 above)
Comment (eliroca) [reply schedule/systemd/suse_patches-systemd_testsuite.yaml:40]: I agree. Let's go for `systemd_testsuite_if_VERSION`

Comment (eliroca) [reply schedule/systemd/suse_patches-systemd_testsuite.yaml:40]: Should that change be in another pull?

**Comment by eliroca** [comment]
> > EDIT: Quotation marks were added: #10152
> 
> I suppose this was already fixed by rebasing, right?

Yep

File: schedule/systemd/suse_patches-systemd_testsuite.yaml (hunk 1 above)
Comment (SergioAtSUSE) [reply schedule/systemd/suse_patches-systemd_testsuite.yaml:40]: > Should that change be in another pull?

No, I think it should be part of the third commit.

Comment (eliroca) [reply schedule/systemd/suse_patches-systemd_testsuite.yaml:40]: Done.

**Pull request merged**